### ISSUE TITLE
[False Positive] "scrollable-region-focusable" with combobox/menu-button and aria-activedescendants patterns

### ISSUE THREAD

**Issue by YuriScarbaci** [dequelabs/axe-core#4914]
### Product

axe-core

### Product Version

4.10.3

### Latest Version

- [x] I have tested the issue with the latest version of the product

#### Expectation
when some specific patterns or arias are present, like aria-activedescendant the scrollable region should not fail, assuming the developer did implement the expected pattern.

#### Actual
the tool laments a non-scrollable region due to scroll coming from javascript

#### How to Reproduce
[menu-button from w3.org/WAI/ARIA/apg/patterns/menu-button with scroll and scrollIntoView()](https://codepen.io/esperiano/full/myVjRGY)

#### Additional context
This is a tricky scenario, but it's a common and valid scenario
this also affect custom application "select" and "combobox" patterns, where the spawing menu contains a list of items that doesn't receive direct focus, only focus styling and are tracked via aria-activedescendent

https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/examples/menu-button-actions-active-descendant/
